client/app/stories/index.jsx

```diff
@@ -0,0 +1,36 @@
+import 'chartjs';
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+
+import Chart from '../bundles/momentDashboards/components/Chart';
```

Comment:
Eventually we'll add module aliasing so that we can just reference "bundles/momentDashboards/components/Chart" or even "momentDashboards/components/Chart" from any path.

Reply:
That is a good point, this will start getting unwieldy pretty soon, but I think we only have 2 components atm so low impact. 

I'm still stuck in the ops side of the PR atm with Circle.CI and Heroku, so I will have to circle back on this (pun intended)